src/app/src/components/Navbar.js

```diff
@@ -73,6 +73,18 @@ const StyledNavbar = styled.div`
     display: ${props => props.hide};
 `;
 
+const getPositionForTab = key => {
+    if (key === 'about') {
+        return 'bottom';
+    } else if (key === 'see') {
+        return 'high';
+    } else if (key === 'meet') {
+        return 'low';
+    } else if (key === 'test') {
+        return 'top';
+    }
+};
```

Comment:
I know this was my suggestion when pairing, but looking at this with fresh eyes I think a function is overkill for this situation.

This would be simpler to maintain/use IMO:
```
const positions = {
  about: 'bottom',
  see: 'high',
  meet: 'low',
  test: 'top',
};
```
And then down below instead of using `setBackgroundPosition(getPositionForTab(key))` we can use `setBackgroundPosition(positions[key])`

Reply:
Sounds good! Implemented this change in 80e5ca3.